internal/fs/inode/dir_test.go

```diff
@@ -1436,21 +1436,31 @@ func (t *DirTest) DeleteChildDir_ImplicitDirTrue() {
 	ExpectFalse(dirIn.IsUnlinked())
 }
 
-func (t *DirTest) CreateLocalChildFile_ShouldnotCreateObjectInGCS() {
-	const name = "qux"
+func (t *DirTest) LocalChildFileCore() {
+	core := t.in.LocalChildFileCore("qux")
 
-	// Create the local file inode.
-	core, err := t.in.CreateLocalChildFile(name)
+	AssertEq(t.bucket.Name(), core.Bucket.Name())
+	AssertEq("foo/bar/qux", core.FullName.objectName)
+	AssertTrue(core.Local)
+}
 
-	AssertEq(nil, err)
-	AssertEq(true, core.Local)
-	AssertEq(nil, core.MinObject)
+func (t *DirTest) InsertIntoTypeCache() {
+	t.in.InsertFileIntoTypeCache("abc")
+	d := t.in.(*dirInode)
 
-	// Object shouldn't get created in GCS.
-	result, err := t.in.LookUpChild(t.ctx, name)
-	AssertEq(nil, err)
```

Comment:
why are we not checking this?

Reply:
Added them back. Initially, it seemed like we are not really doing anything while creating the core except mapping it to a Core object. So, wasn't sure if we should test all these flows, but now I think it makes sense.